fiduceo_uncertainties.f90

```diff
@@ -784,16 +790,41 @@ SUBROUTINE Write_Temp_NETCDF(temp_file,AVHRR,FCDR,twelve_micron_there)
        call check(stat)
     ENDIF
 
+!MT: 03-11-2017: fix problem of value not filling array     
+!    ALLOCATE(ch1_non_random(nx,ny), stat)
+!    ALLOCATE(ch2_non_random(nx,ny), stat)
+!    ALLOCATE(ch3a_non_random(nx,ny), stat)
+!    IF( stat.ne.0 )THEN
+!       error
+!    ENDIF
+!    ch1_non_random = -1e30
+!    ch2_non_random = -1e30
+!    ch3a_non_random = -1e30
+!    WHERE(ch1_varid.ne.-1e30)ch1_non_random=0.03
+!    WHERE(ch2_varid.ne.-1e30)ch2_non_random=0.05
+!    WHERE(ch3a_varid.ne.-1e30)ch3a_non_random=0.05
+!    DEALLOCATE(ch1_non_random)
```

Comment:
You should only deallocate the arrays once they have been used - which at this point they haven't been

Reply:
Yes, thank you.

--> Retain comment of DEALLOCATE(ch1_non_random...)